### PR TITLE
refactor(frontend): new descriptive hyperlink ENG #523

### LINKED ISSUE #523: Change "link" by descriptive words
Change the word "link" from guides by descriptive words about the link goal 
Example:
imagen

If you want know all cases, can use the browser
imagen

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- resolves #523 

### Steps to test
1. Go to browser in VSCode and search "[link]"

#### CheckList
- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: docs/eos-learn/eosio-components.md [hunk 1]
```diff
@@ -123,6 +123,6 @@ Installing eosio.cdt will make the compiled binary global so, it can be accessib
 ```shell
 $ sudo ./install.sh
 ```
->For more information on how to Install the Contract DEV. Toolkit go to [this page](https://developers.eos.io/welcome/latest/getting-started/development-environment/install-the-CDT).
+>For more information on how to Install the Contract DEV. Toolkit go to guide[How Install CDT](https://developers.eos.io/welcome/latest/getting-started/development-environment/install-the-CDT).
```
Comment (Torresmorah) [inline docs/eos-learn/eosio-components.md:126]: Add a space

**Review by Torresmorah — COMMENTED** [review]
In my opinion, there are some links that could be more descriptive, for example, "click here", "more information", and "our guide"

File: docs/eos-learn/important-functions.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Usually this authority is invoked by a multi-signature (or "multisig") contract
...
+EOSIO smart contracts are simple WebAssembly files. The WebAssembly format allows you to define imports, functions that are not declared in WASM itself but in the environment in which the WebAssembly code is run. EOSIO uses these built-in (intrinsic) functions to exchange data between the WebAssembly contract code and the EOSIO node `nodeos`. Every time the EOS virtual machine (EOSVM), which runs the WebAssembly code, reaches an intrinsic level, control of the program is transferred to the node controller code to resolve it. A complete list of intrinsic functions for EOSIO 2.0.X can be found [Intrinsic Functions](https://github.com/EOSIO/eosio.cdt/blob/a6b8d3fc289d46f4612588cdd7223a3d549238f6/libraries/native/native/eosio/intrinsics_def.hpp#L42-L160).
```
Comment (xavier506) [inline docs/eos-learn/important-functions.md:17]: can be found here : [Intrinsic Functions]...

**Review by xavier506 — APPROVED** [review]
Great work ! 💯

File: docs/eos-learn/important-functions.md (hunk 2 above)
Comment (JAMares) [reply docs/eos-learn/important-functions.md]: Fixed up

File: docs/consensus-mechanisms-blockchain.md [hunk 3]
```diff
@@ -89,4 +89,4 @@ This mechanism is born with a very old proposal, but different from the previous
 This mechanism was born as an Intel solution to the advantage obtained by those with more resources when looking for the fairest block validator election system, like a lottery within the network.
 
 
-If you want to learn more about the different consensus mechanisms, you can visit the following [link](https://www.allerin.com/blog/8-blockchain-consensus-mechanisms-you-should-know-about).
+If you want to learn more about the different consensus mechanisms, you can visit the following [consensus mechanisms you should know about](https://www.allerin.com/blog/8-blockchain-consensus-mechanisms-you-should-know-about).
```
Comment (xavier506) [inline docs/consensus-mechanisms-blockchain.md:92]: visit the following **page** [consensus mechanisms you should know about]

Comment (JAMares) [reply docs/consensus-mechanisms-blockchain.md:92]: Fixed up

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**